resources/js/frontend/helper/wc-payplug-apple_pay-requests.js

```diff
@@ -43,3 +42,64 @@ export const apple_pay_update_payment = (data) => {
 		});
 	});
 }
+
+export const apple_pay_UpdateOrder = (data) =>{
+	return new Promise((resolve, reject) => {
+		$.ajax({
+			type: 'POST',
+			data: data,
+			url: settings.ajax_url_update_applepay_order
+		}).success(function (response) {
+			resolve(response);
+
+		}).error(function (xhr, status, error) {
+			reject(error); // NOT WORKING!!
+		});
+	});
+}
+
+export const apple_pay_Payment = (data) =>{
+	return new Promise((resolve, reject) => {
+		$.ajax({
+			type: 'POST',
+			data: data,
+			url: settings.ajax_url_update_applepay_payment
+		}).success(function (response) {
+			resolve(response);
+
+		}).error(function (xhr, status, error) {
+			reject(error); // NOT WORKING!!
+		});
+	});
+}
+
+export const apple_pay_PlaceOrderWithDummyData = (data) =>{
+	return new Promise((resolve, reject) => {
+		$.ajax({
+			type: 'POST',
+			data: data,
+			url: settings.ajax_url_place_order_with_dummy_data
+		}).success(function (response) {
+			resolve(response);
+
+		}).error(function (xhr, status, error) {
+			reject(error); // NOT WORKING!!
+		});
+	});
+}
+
+export const apple_pay_CancelOrder = (data) =>{
+	return new Promise((resolve, reject) => {
+		$.ajax({
+			type: 'POST',
+			data: data,
+			url: settings.ajax_url_applepay_cancel_order
+		}).success(function (response) {
+			resolve(response);
+
+		}).error(function (xhr, status, error) {
+			reject(error); // NOT WORKING!!
+		});
+	});
+}
```

Comment:
In order to keep the codebase simple and SOLID wouldn't be better
```suggestion
export const apple_pay_Post = (data, url) =>{
	return new Promise((resolve, reject) => {
		$.ajax({
			type: 'POST',
			data: data,
			url
		}).success(function (response) {
			resolve(response);

		}).error(function (xhr, status, error) {
			reject(error); // NOT WORKING!!
		});
	});
}
```